spark/v3.4/spark/src/test/java/org/apache/iceberg/spark/source/TestIcebergSourceTablesBase.java

```diff
@@ -240,8 +237,12 @@ public void testEntriesTablePartitionedPrune() {
             .select("status")
             .collectAsList();
 
-    Assert.assertEquals("Results should contain only one status", 1, actual.size());
-    Assert.assertEquals("That status should be Added (1)", 1, actual.get(0).getInt(0));
+    assertThat(actual)
+        .singleElement()
+        .satisfies(
+            row -> {
```

Comment:
nit: {} can be removed here

Reply:
Thanks for checking this, let me update this.